functions-methods.js

```diff
@@ -9,13 +9,7 @@
 // getEmailDomain("t.mellink@novi.nl") geeft novi.nl
 // getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
 
-function getEmailDomain(email, domain) {
-    return email.slice("@") === domain;
-}
 
```

Comment:
Hier komt true of false uit en geen domain. Verder kun je slice wel gebruiken. In uitwerkingen gebruiken ze substring.
Ik heb split functie gebruikt.